Add API Docs for showing a Team Token (#703)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/team-tokens.mdx

  ]
}
```

> ## Show a Team Token

Reviewer: Since we use "List" as the verb above, how would you feel about tweaking to match that language:
```suggestion
## List a team's token
```
Also, could you do me a huge favor and rewrite the headers in this file to use sentence case to match our style guide? 🙏

Author: Yes, can do on the sentence case!

Is  `List` the right verb here though? Generally in the API docs "List" means "show all the resources of this type" and "Show" means "show a single resource, with more detail". E.g. [Show and Organization](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organizations#show-an-organization)
We actually already have a "list" for this page https://developer.hashicorp.com/terraform/cloud-docs/api-docs/team-tokens#list-team-tokens

> A [Team Token](/terraform/cloud-docs/users-teams-organizations/teams#api-tokens) can be displayed for a particular team:

Reviewer: Quick suggestions for active voice and capitalization (we only capitalize product names):
```suggestion
You can display a [team token](/terraform/cloud-docs/users-teams-organizations/teams#api-tokens) for a particular team using:
```

`GET /teams/:team-id/authentication-token`

| Parameter  | Description               |
| ---------- | ------------------------- |
| `:team-id` | The ID of the Team.       |

> or fetched by its ID:

Reviewer: Adding a bit more context around this table for clarity. Please feel free to pushback if this doesn't feel good though!
```suggestion
You can also fetch a team's tokens by using the team's ID with the `authentication-tokens/` endpoint:
```

Author: this is actually not quire correct, it is the token's ID, not the team's ID which is being used.

Would this work?
```suggestion
You can also fetch a team tokens directly by using the token's ID with the `authentication-tokens/` endpoint:
```

Reviewer: That works great, thank you for the context and correction!

Reviewer: Once you add your suggestion please feel free to merge this too! 🔥
